rc/windowing/iterm.kak

```diff
@@ -29,7 +29,7 @@ define-command -hidden -params 2.. iterm-terminal-impl %{
         escaped=$(do_esc "$args")
         esc_path=$(do_esc "$PATH")
         esc_tmp=$(do_esc "$TMPDIR")
-        cmd="env PATH='${esc_path}' TMPDIR='${esc_tmp}' $escaped"
+        cmd=$(do_esc "/bin/sh -c \"exec env PATH='${esc_path}' TMPDIR='${esc_tmp}' $escaped\"")
```

Comment:
is this really about PATH or is it the size of the command line or environment?
I'm not sure what's going on here.

Commit 3ae94d017 suggests that iterm launches the user's default shell, hence the use of `env`. Maybe this problem is specific to the default shell and somehow /bin/sh is not affected?

Reply:
If I wrap this with any of /bin/sh -c, /bin/zsh -c, /run/current-system/bin/zsh -c (my interactive shell), the issue does not occur, so doesn't look like it passes it verbatim to any shell.